src/theming/Helpers/arrayRange.ts

```diff
@@ -0,0 +1,3 @@
+export function arrayRange (size: number) {
+  return [...(new Array(size))].map((_, index) => index)
```

Comment:
`new Array()` is not working in build.

```suggestion
  return Array.from({ length: size }, (_, index) => index)
```